src/domain/children/ChildShow.tsx

```diff
@@ -77,9 +80,12 @@ const ChildShow = () => {
             label="children.fields.occurrences.label"
           >
             <Datagrid bulkActionButtons={false} rowClick={onClickOccurrence}>
-              <DateField
+              <FunctionField
                 label="occurrences.fields.time.fields.date.label"
                 source="node.time"
+                render={(record?: Partial<OccurrenceNodeEdge>) =>
+                  record?.node ? toDateString(new Date(record.node?.time)) : ''
+                }
```

Comment:
```suggestion
                render={(record?: Partial<OccurrenceNodeEdge>) =>
                  record?.node
                    ? toDateString(new Date(record.node?.time), locale)
                    : ''
                }
```

Just adding the `locale` parameter to `toDateString` call & linter autoformatted the result.